docs/wallet-integration.md

```diff
@@ -0,0 +1,126 @@
+## Wallet Integration
```

Comment:
I recommend this guide is structured as follows:

Two ways to integrate a wallet:

1. Browser extension wallets: <Some text explaining that wallets should be integrated through Tomo Connect and that native integrations happen infrequently only for critical wallets for Babylon (see my above text)>
2. Mobile wallets: <Text explaining the interfaces that should be injected>

Some stuff is not needed, see individual comments below.